back-end/API/app/repositories/enterprise_repo.py

```
@@ -53,15 +53,21 @@ def getEnterprise(self, id):
+        try:
+            enterprise = Enterprise.query.filter_by(id=data['id']).first()
+            if enterprise:
+                enterprise.name = data['name']
+                enterprise.email = data['email']
+                enterprise.phone = data['phone']
+                enterprise.address = data['address']
+                enterprise.cityId = data['cityId']
+                db.session.commit()
+                return enterprise
+            else:
+                raise NotFoundException("Enterprise not found")
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)